Cargo.toml

```diff
@@ -12,6 +12,7 @@ members = [
   "src/tools/bump-stage0",
   "src/tools/cargotest",
   "src/tools/clippy",
+  "src/tools/clippy/clippy_test_deps",
```

Comment:
@RalfJung I added this to the `Cargo.toml` to make the tests pass. Is this correct?

Reply:
Hm, that seems odd, I don't think we did that for Miri. It should probably be excluded from the workspace, so that it can have its own lockfile checked in.

Reply:
But for Miri we didn't have to manually exclude anything.

Reply:
Ok, I'll look into this tomorrow. Does `miri` have a Cargo.lock file checked in?